erpnext/public/js/help_links.js

```diff
@@ -1,11 +1,35 @@
 frappe.provide('frappe.help.help_links');
 
 frappe.help.help_links['Form/Rename Tool'] = [
-	{ label: 'Bulk Rename', url: 'https://frappe.github.io/erpnext/user/manual/en/setting-up/data/bulk-rename.html' },
+	{ label: 'Bulk Rename', url: 'https://frappe.github.io/erpnext/user/manual/en/setting-up/data/bulk-rename' },
 ]
 
-frappe.help.help_links['List/Customer'] = [
-	{ label: 'Customer', url: 'https://frappe.github.io/erpnext/user/manual/en/CRM/customer.html' },
+//Setup
+
+frappe.help.help_links['List/User'] = [
+	{ label: 'New User', url: 'httphttps://frappe.github.io/erpnext/user/manual/en/setting-up/users-and-permissions/adding-users' },
```

Comment:
should be `https://frappe.github.io/erpnext/user/manual/en/setting-up/users-and-permissions/adding-users`